content/1.7/back-office/shop-parameters/search/search-listing.md

```diff
@@ -0,0 +1,179 @@
+Below, the following message is displayed : "You can set a cron job that will rebuild your index using the following URL:" 
+
+A link to the AdminSearch controller is displayed: "http://localhost/presta1781/admin/index.php?controller=AdminSearch&action=searchCron&ajax=1&full=1&token=yruOjDwR&id_shop=1" (the link depends on the shop domain name and admin folder's name)
+Below, a toggle "Indexing" allows to enable or disable the automatic indexing of products.
+The following help text is displayed: "Enable the automatic indexing of products. If you enable this feature, the products will be indexed in the search automatically when they are saved. If the feature is disabled, you will have to index products manually by using the links provided in the field set."
+By default the toggle is set to "yes".
+
+ # Search
+ 
+Firstly, an information message is displayed: "We are thrilled to introduce you to the fuzzy search, one of the new features from 1.7.7! Please note that it is still in beta version, so feel free to share improvement ideas on GitHub to have it enhanced."
+Below there are 2 links:
+ - [Signaler un problème sur GitHub](https://github.com/PrestaShop/PrestaShop/issues/new/choose)
+ - [Proposer une idée d'amélioration sur GitHub](https://github.com/PrestaShop/PrestaShop/issues/new/choose)
+Below, the following toggle and fields are displayed:
+  - "Search within word" : This toggle, can be enabled or disabled. The following help text is displayed: "By default, to search for “blouse”, you have to enter “blous”, “blo”, etc (beginning of the word) – but not “lous” (within the word). With this option enabled, it also gives the good result if you search for “lous”, “ouse”, or anything contained in the word.". 
+The following tooltip is displayed: "Enable search within a whole word, rather than from its beginning only. It checks if the searched term is contained in the indexed word. This may be resource-consuming."
+By default the toggle is set to "No".
+ - "Search exact end match" toggle switch, can be turned to "yes" or "no". Has an explanation below: "By default, if you search "book", you will have "book", "bookcase" and "bookend". With this option enabled, it only gives one result “book”, as exact end of the indexed word is matching." 
+The following tooltip is displayed: "Enable more precise search with the end of the word. It checks if the searched term is the exact end of the indexed word."
+By default the toggle is set to "No".
+ - "Fuzzy search": This toggle, can be enabled or disabled. Has an explanation below: "By default, the fuzzy search is enabled. It means spelling errors are allowed, e.g. you can search for "bird" with words like "burd", "bard" or "beerd". Disabling this option will require exact spelling for the search to match results." 
+The following tooltip is displayed: "Enable approximate string matching".
+By default this toggle is set to "Yes".
```

Comment:
```suggestion

```

Reply:
It's more logical to put this at the beginning when you say that it can be enabled or disabled.